core/repository/block/headers.ts

```diff
@@ -0,0 +1,5 @@
+import { BlockId } from 'ddk.registry/dist/model/common/type';
+
+import { BlockHeaders } from 'shared/model/headers/block';
+
+export const BlockHeadersRepository = new Map<BlockId, BlockHeaders>();
```

Comment:
this is not repository :)

Reply:
BlockHeadersStorage

Reply:
Why not? It produces methods for setting, getting, deleting, etc.

Reply:
What if you need custom logic. For example, `give me headers with those Ids`

Reply:
When I will create a class with this logic, but now this is not necessary